src/core/display/Container.js

```diff
@@ -544,3 +544,12 @@ Container.prototype.renderCanvas = function (renderer)
         renderer.maskManager.popMask(renderer);
     }
 };
+
+/**
+ * Destroys the container
+ *
+ */
+Container.prototype.destroy = function ()
+{
+    this.children = null;
```

Comment:
Hey guys. I think we could add a 'destroyChildren' property to the function (much like the destroy texture function) That we we have both angles covered.